<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Func</class>
 <widget class="QDialog" name="Dialog_Func">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Func:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SmartTextEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartTextEdit</class>
   <extends>QTextEdit</extends>
   <header>comp.smarttextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_Func</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_Func</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
